README.md

```diff
@@ -21,8 +21,9 @@ Read the blog post about this repository at:
 ## Installation
 
 ```
-$ composer require broadway/broadway ~0.2.0
+$ composer require broadway/broadway ~0.5.2
```

Comment:
Could you change it to just `composer require broadway/broadway`? Composer uses the latest version by default then. Saves us updating this regularly :)